dist/google-music.js

```diff
@@ -279,6 +293,16 @@ proto.rating = {
     if (el && !this.rating._isElSelected(el)) {
       el.click();
     }
+  },
+
+  // Reset the rating
```

Comment:
Was trying to be consistent with the other function around it who all appear to have this style of comment above them?  Please confirm that I should remove this one @twolfson